_maps/bubber/away_missions/big_wargames.dm

```diff
@@ -0,0 +1,125 @@
+/area/awaymission/wargames
```

Comment:
move this file to somewhere more sane, an actual code folder, putting it this far up in the load order breaks the entire game

Reply:
oopsies.